src/libraries/System.Text.Json/gen/Resources/xlf/Strings.ja.xlf

```diff
@@ -22,6 +22,16 @@
         <target state="translated">重複した種類名。</target>
         <note />
       </trans-unit>
+      <trans-unit id="MultipleJsonConstructorAttributeFormat">
```

Comment:
Out of curiosity, did you make these additions manually or were they generated using some tool?

Reply:
They were generated using this Xliff localization tool - https://github.com/dotnet/runtime/blob/c16cb0ed11d6473192fb7dfc76245404a2e8b3ef/src/libraries/System.Text.Json/gen/System.Text.Json.SourceGeneration.csproj#L9.